extensions/azure-openai-oauth-rshiny/manifest.json

```diff
@@ -0,0 +1,1862 @@
+{
+  "version": 1,
+  "locale": "en_US",
+  "platform": "4.3.0",
+  "metadata": {
+    "appmode": "shiny",
+    "primary_rmd": null,
+    "primary_html": null,
+    "content_category": null,
+    "has_parameters": false
+  },
+  "extension": {
+    "name": "azure-openai-oauth-rshiny",
+    "title": "R Shiny Azure OpenAI OAuth Integration Example",
+    "description": "An R Shiny app with an Azure OpenAI OAuth Integration that answers questions about the Palmer Penguins dataset",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/azure-openai-oauth-rshiny",
+    "category": "example",
+    "tags": ["r", "shiny"],
+    "minimumConnectVersion": "2025.06.0",
```

Comment:
For my own edification is there a specific oauth integration added in `2025.06.0` that this requires or something else that makes this the minimum Connect version.